x-pack/plugins/observability_solution/observability/public/pages/alerts/alerts.tsx

```diff
@@ -72,6 +74,31 @@ function InternalAlertsPage() {
 
   const filteredRuleTypes = useGetFilteredRuleTypes();
 
+  const { setScreenContext } = observabilityAIAssistant?.service || {};
+
+  const ruleTypesWithDescriptions = useGetAvailableRulesWithDescriptions();
+
+  useEffect(() => {
+    return setScreenContext?.({
+      data: ruleTypesWithDescriptions.map((rule) => ({
+        name: rule.id,
+        value: `${rule.name} ${rule.description}`,
```

Comment:
you can use `value: rule`. In this case it probably is overkill to use `data` (I wasn't aware of the shape, sorry). But it's also fine to leave it as-is, there's not a lot of tokens so it gets sent over automatically.